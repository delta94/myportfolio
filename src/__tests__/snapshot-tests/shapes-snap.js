import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Shapes from '../../components/shapes';
import HeroImage from '../mocks/hero-image.json';

it('Renders Shape correctly', () => {
  const wrapper = shallow(<Shapes heroImage={HeroImage} />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
