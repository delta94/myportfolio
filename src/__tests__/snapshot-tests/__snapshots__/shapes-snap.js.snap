// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Shape correctly 1`] = `
<styled.div
  className="is-hidden-touch"
>
  <ShapeView
    key="Skills"
    name="Skills"
    type={1}
  >
    <Image
      Tag="div"
      alt="hero-image-mirror"
      className="shape-image-wrapper"
      fadeIn={true}
      outerWrapperClassName="shape-image-outer-wrapper"
      sizes={
        Object {
          "sizes": Object {
            "aspectRatio": 1.48720999405116,
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAwACBf/EABUBAQEAAAAAAAAAAAAAAAAAAAEC/9oADAMBAAIQAxAAAAEtc9YtYYf/xAAbEAACAgMBAAAAAAAAAAAAAAABAgASESEiMf/aAAgBAQABBQKy4uY2yDC3B9//xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAwEBPwEn/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAgEBPwE//8QAGhAAAgIDAAAAAAAAAAAAAAAAAAEQMQIhYf/aAAgBAQAGPwK8myzbh8j/xAAaEAEBAQEAAwAAAAAAAAAAAAABACERMXGR/9oACAEBAAE/IeukvA4Fhse+k3yQT0j13//aAAwDAQACAAMAAAAQ1+//xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAwEBPxAAT//EABYRAQEBAAAAAAAAAAAAAAAAAAEQEf/aAAgBAgEBPxB1Z//EABwQAQACAwADAAAAAAAAAAAAAAERIQBBUTGBwf/aAAgBAQABPxCcTBAHKvmXdYRkIKLw1Jpm5ek5BMsQ7MvyPeEgobDmf//Z",
            "sizes": "(max-width: 2500px) 100vw, 2500px",
            "src": "/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-2b4d0.jpg",
            "srcSet": "/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-98cf1.jpg 700w,
/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-90fd0.jpg 1400w,
/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-2b4d0.jpg 2500w",
          },
        }
      }
    />
  </ShapeView>
</styled.div>
`;
